app/scripts/directives/schoollist.coffee

```diff
@@ -20,12 +20,13 @@ angular.module 'edudashApp'
       property: '@property'
       max: '=max'
       min: '=min'
```

Comment:
@esoliani if you agree, I think we should use the `@property` with the `bracketsSrv` service to colourize, instead of passing in `min` and `max` here and doing the logic in the view.

This can be in a separate PR though, and I can do it if you want.